From a7d44d68643167dda5ecc139c0992f4efaf05997 Mon Sep 17 00:00:00 2001
From: Xi Ruoyao <xry111@xry111.site>
Date: Wed, 8 Apr 2026 11:09:18 +0800
Subject: [PATCH 12/12] AOSCOS: drop redundant and incorrect systemEnv()
 declaration

The header dm.h has already declared systemEnv() so it's redundant to
declare it again in Greet.c.  And the declaration in Greet.c is
incorrect in ISO C >= 2017.

Signed-off-by: Xi Ruoyao <xry111@xry111.site>
---
 src/wdm/Greet.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/wdm/Greet.c b/src/wdm/Greet.c
index 782620f..e27321e 100644
--- a/src/wdm/Greet.c
+++ b/src/wdm/Greet.c
@@ -140,8 +140,6 @@ static char     xsessionArg[256], exitArg[256];
 
 struct display *Save_d=NULL;
 
-extern  char    **systemEnv();
-
 /****  pipe I/O routines ****/
 
 /** The following code was adapted from in.c by Tom Rothamel */
-- 
2.53.0

